Source/Krypton Components/TestForm/FadeFormTest.cs

```diff
@@ -7,13 +7,71 @@
  */
 #endregion
 
+using System.Threading;
+
 namespace TestForm
 {
     public partial class FadeFormTest : KryptonForm
     {
         public FadeFormTest()
         {
             InitializeComponent();
+            cbtnShowImage.Checked = true;
+        }
+
+        private void FadeFormTest_FormClosing(object sender, FormClosingEventArgs e)
```

Comment:
So this is a workaround just for the test - app ?
Doesn;t solve the usage of the FadeForm at it's core at all !

Reply:
> So this is a workaround just for the test - app ? Doesn;t solve the usage of the FadeForm at it's core at all !
Doesn't resolve it yet. 
This way you both can test this and see what works / give good results.
From there on the controller routines can be adjusted as a final solution.